features/ubuntu_upgrade.feature

```diff
@@ -50,8 +50,6 @@ Feature: Upgrade between releases when uaclient is attached
       | bionic  | lxd-container | focal        | lts    |               | esm-infra | enabled         | esm-apps | enabled         | true           |
       | bionic  | lxd-container | focal        | lts    |               | usg       | enabled         | usg      | enabled         | pro enable cis |
       | focal   | lxd-container | jammy        | lts    |               | esm-infra | enabled         | esm-apps | enabled         | true           |
-      | jammy   | lxd-container | mantic       | normal |               | esm-infra | n/a             | esm-apps | n/a             | true           |
```

Comment:
We need a test that tests an upgrade from jammy. If it now upgrades to `noble`, then we need to update this line, not remove it

Reply:
I did run the upgrade from `jammy` to `noble` but it failed and there was a comment right below which comments out the jammy -> noble upgrade.
 
 ```
# No path from Jammy to Noble until .1 is there
# | jammy   | lxd-container | noble        | lts    |                 | esm-infra | enabled         | esm-apps | enabled         | true           |
 ```

Reply:
Sorry - now I remember. This is okay to delete

We should probably have the jammy->noble line uncommented but with the `--devel-release` flag - I'll add a separate comment with that suggestion.

Reply:
devel release won't work anymore, because devel is now oracular. Commented is good.

Reply:
LTS' are weird prior to next LTS.1
- `Prompt=lts` says "There is no development version of an LTS available."
- `Prompt=normal` + `--devel-release` looks at oracular and says "Upgrades to the development release are only available from the latest supported release."
- `Prompt=lts` + `--devel-release` upgrades successfully no noble :upside_down_face:

Reply:
alright good good